common.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	getKeyvaluestoreEndpoint: function(){
+		if (process.argv.length < 3 && !process.env.MICRO_KEYVALUESTORE_ENDOPOINT) {	    
```

Comment:
Replace `MICRO_KEYVALUESTORE_ENDOPOINT` with `MICRO_KEYVALUESTORE_ENDPOINT` in all modified scripts.